### ISSUE TITLE
oab-java.sh (0.3.0) fails in current state

### ISSUE THREAD

**Issue by brendandc** [flexiondotorg/oab-java6#123]
I get the following error in the log:
cp -a x64/jre debian/oracle-java7-bin/usr/lib/jvm/java-7-oracle-1.7.0.51
cp: cannot stat `x64/jre': No such file or directory
make: **\* [binary-oracle-java7-bin] Error 1
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
^H^H8242's retcode: 2
success

And this in standard out:
ERROR! Packages failed to build.

Somebody else filed: https://github.com/rraptorr/oracle-java7/issues/15 against oracle-java7 but they think its an issue here. It seemed like they had to change their format due to changes in dpkg (see comment in link).

For reference I am running the script on vagrant's precise64.box. Any other details, I'd be happy to provide.

**Comment by shinji257** [comment]
Just a heads up but I believe #119 is an older report of this issue.

**Comment by x3bandy** [comment]
I've somewhat similar issue regardless of the architecture

cp -a x64/jre debian/oracle-java7-bin/usr/lib/jvm/java-7-oracle-1.7.0.51
cp: cannot stat `x64/jre': No such file or directory
make: **\* [binary-oracle-java7-bin] Error 1
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
^H^H29243's retcode: 2

cp -a i586/jre debian/oracle-java7-bin/usr/lib/jvm/java-7-oracle-1.7.0.51
cp: cannot stat `i586/jre': No such file or directory
make: **\* [binary-oracle-java7-bin] Error 1
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
9696's retcode: 2

**Comment by shinji257** [comment]
Try the referenced pull request an see if it works for you when manually applied.  Hopefully the maintainer will accept the patch.

**Comment by x3bandy** [comment]
Thanks!

I can confirm that it works

**Comment by drdamour** [comment]
yup worked for me 

```
wget https://raw.github.com/ladios/oab-java6/70408f49e9e5b44820354b630220ca2c0c1060a7/oab-java.sh -O oab-java.sh
```

for now...

**Comment by jsirex** [comment]
Unable to make it work :(
Tried different versions of oab-java.sh. Also, from ladios.
Tried script mentiod by @drdamour 
OS: Debian Jessie

Using rraptorr/oracle-java7 directly works.
It seems that something wrong with paths.

UPD: I've used master of rraptorr/oracle-java7

**Comment by jsirex** [comment]
Update to previous post: recently make it works with branch from ladios/master

**Comment by zasran** [comment]
Yeah, same as jsirex, https://github.com/ladios/oab-java6/blob/master/oab-java.sh works with Java 8. Any chances this repository will be updated?
